src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Text, SafeAreaView, StyleSheet } from 'react-native';
+
+import { theme } from '../constants';
+const { SIZES, FONTS, COLORS } = theme;
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file and about the style files.